frontend/src/metabase-lib/queries/StructuredQuery.ts

```diff
@@ -1719,6 +1746,7 @@ class StructuredQuery extends memoizeClass<StructuredQueryInner>(
   "joinedDimensions",
   "breakoutDimensions",
   "aggregationDimensions",
+  "aggregationOperatorsLookup",
```

Comment:
as this function is memoized in `Table`:
https://github.com/metabase/metabase/blob/fa37b582653012264173e72a786a99a312eef345/frontend/src/metabase-lib/metadata/Table.ts#L176

I should do that here as well to stay consistent.